src/main/java/pipe/gui/GuiFrame.java

```diff
@@ -232,17 +232,17 @@ public void actionPerformed(ActionEvent arg0) {
             currentTab.ifPresent(TabActions::verifySelectedQuery);
         }
     };
-    private final GuiAction workflowDialogAction = new GuiAction("Workflow analysis", "Analyse net as a TAWFN", KeyStroke.getKeyStroke(KeyEvent.VK_F, shortcutkey)) {
+    private final GuiAction workflowDialogAction = new GuiAction("Workflow analysis", "Analyses net as a TAWFN", KeyStroke.getKeyStroke(KeyEvent.VK_F, shortcutkey)) {
```

Comment:
There should be Analyse, not Analyses

Reply:
OK. I wanted to be consistent with the other tooltips, but I see why I shouldn't